experimental/fast_lossless/fast_lossless.cc

```diff
@@ -1324,9 +1328,16 @@ size_t LLEnc(const unsigned char* rgba, size_t width, size_t stride,
   return writer.bytes_written;
 }
 
-size_t FastLosslessEncode(const unsigned char* rgba, size_t width,
-                          size_t stride, size_t height, size_t nb_chans,
-                          size_t bitdepth, int effort, unsigned char** output) {
+}  // namespace
+
+size_t JxlFastLosslessEncode(const unsigned char* rgba,
```

Comment:
doesn't this need an extern "C" too then?

Reply:
Technically, I don't think it's necessary (https://stackoverflow.com/a/1380926). But I put an extern here too.

Reply:
uh, TIL. Still useful to have it for documentation and possible shenanigans.